### PR TITLE
[COST-95] README updates

### PR DESCRIPTION
**Pull request by maskarb**
related #247

### CODE REVIEW

File: docs/cost_usage_report_generation.rst [hunk 1]
```diff
@@ -0,0 +1,181 @@
+
+Example ``nise report`` usage
+=============================
+
+nise is a command line tool::
```
Comment (abaiken) [inline docs/cost_usage_report_generation.rst:5]: Do we need to have the usage in both the README & the cost_usage_report_generation? If we do - is there a reason that they are different?

Comment (maskarb) [reply docs/cost_usage_report_generation.rst:5]: The README contains the `yaml` usage and `report` usage options, whereas in the `cost_usage_report_generation` I only put the report options and in the `yaml_generation` I only have the yaml options (if they are different beyond that, def let me know. I might be going crazy looking at these files). I just wanted the example readme files to show whatever options are actually pertinent info to what the user is trying to do.

Comment (abaiken) [reply docs/cost_usage_report_generation.rst:5]: Ahhhh! That makes sense. Thanks for the clarification!

**Comment by maskarb** [comment]
Merged into #247

**Pull request closed without merging**